public/_includes/landing-promo-1.ejs

```diff
@@ -2,7 +2,8 @@
   <div class="features-wrap">
     <div class="features-list"><!-- class="media-description" -->
       <header class="features-item">
-      <h2>Which <code>WebView</code> are you choosing?</h2>
+      <h2 lang="en">Which <code>WebView</code> are you choosing?</h2>
+      <h2 lang="zh">选择哪一个<code>WebView</code>？</h2>
```

Comment:
It is not good for “选择哪一个”， because there is only Crosswalk WebView been introduced and no comparason. But I've no idea the better one. how about "需要什么样的WebView"? @chuan9 @lisa0314 what do you think?